controllers/articlesReactionController.js

```diff
@@ -0,0 +1,98 @@
+const {
+  getUserReaction,
+  createReaction,
+  updateReaction,
+  removeReaction,
+  getTotalReactions
+} = require('../lib/modelManagers/articlesReactionModel');
+
+const {
+  SERVER_ERROR_MESSAGE,
+  RESET_REACTION,
+  RESOURCE_CREATED_CODE,
+  ARTICLE_REACTION_STATUS,
+  OK_CODE
+} = require('../constants');
+const {
+  successResponse,
+  serverFailure,
+  formatReaction,
+  formatMessage
+} = require('../lib/utils/messageHandler');
+
+let newReaction;
+/**
+ *
+ *
+ * @class commentsReactionController
+ */
+class ArticlesReactionController {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @memberof ArticlesReactionController
+   * @returns {object} response of the request
+   */
+  static async allReactions(req, res) {
+    const message = formatMessage(req.params.reaction);
+    try {
+      const reactions = await getTotalReactions(req.params.articleId, req.params.reaction);
```

Comment:
`getTotalReactions` This name feels more like we are getting only the total number of reactions. If your interest here is just the total reactions, you should use the count method in sequelizejs.
http://docs.sequelizejs.com/manual/tutorial/models-usage.html#-count-count-the-occurrences-of-elements-in-the-database